### PR TITLE
admin bugs resolved

### PR DESCRIPTION
**Pull request by Deepakkumar-kaaylabs**

### CODE REVIEW

File: src/components/generics/modal-component.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ const ModalComponent = (props: Props) => {
   return (
     <Modal className="lookup-modal" show={open} onHide={props.onCancel}>
       <Modal.Header className="lookup-modal-header" closeButton>
-        {title && <Modal.Title>{title}</Modal.Title>}
+        {!!title && <Modal.Title>{title}</Modal.Title>}
```
Comment (MJyotheesh) [inline src/components/generics/modal-component.tsx:66]: !!title or title both are same right

File: src/pages/driver/index.tsx [hunk 2]
```diff
@@ -68,16 +68,18 @@ const Driver = (): JSX.Element => {
...
-      <div className="title p-2">
+      <div className="title p-2 pb-0 mx-2">
         <TitleSearch
           placeholder="Driver Search"
           title="Driver"
-          onChange={(value) => { filterData(value); }} // need to do search feature
+          onChange={(value) => {
+            filterData(value);
+          }}
           className=""
```
Comment (MJyotheesh) [inline src/pages/driver/index.tsx:78]: what is the purpose of empty className?

File: src/pages/supplier/supplier-profile.tsx [hunk 3]
```diff
@@ -22,7 +20,7 @@ const SupplierProfile = (props: SupplierProfileProps): JSX.Element => {
     try {
       setLoading(true);
       const res = await getSupplier({ supplierUid, userType: userRole });
-      const result = get(res,"result",[{}]);
+      const result = get(res, "result", [{}]);
```
Comment (MJyotheesh) [inline src/pages/supplier/supplier-profile.tsx:23]: give []

**Review by MJyotheesh — CHANGES_REQUESTED** [review]
Resolve comments

File: src/components/generics/modal-component.tsx (hunk 1 above)
Comment (Deepakkumar757) [reply src/components/generics/modal-component.tsx:66]: removed

File: src/pages/driver/index.tsx (hunk 2 above)
Comment (Deepakkumar757) [reply src/pages/driver/index.tsx:78]: removed

**Review by MJyotheesh — APPROVED** [review]

**Pull request merged**